File: QC_qns/swap_twonumbers.c
#include<stdio.h>
int main()
{
	int a,b;
	printf("enter a number:");
	scanf("%d",&a);
	printf("enter b number:");
	scanf("%d",&b);
	printf("before swapping the numbers are a:%d\tb:%d\n",a,b);
	a = a ^ b;
	b = a ^ b;
	a = a ^ b;
	printf("after swapping the numbers are \n");
	printf("a=%d\tb=%d\n",a,b);
}

File: python/functions/swap_twonumbers.py
def swap_fun(m,n):
    m = m ^ n
    n = m ^ n
    m = m ^ n
    print("after swapping the two numbers is")
    print("m=",m,"\tn=",n)

a=int(input("enter a number:"))
b=int(input("enter b number:"))
print("before swap two numbers are a=",a  ,"b=",b)
swap_fun(a,b)
